mocks/log.go

```diff
@@ -126,47 +126,47 @@ func (msw *MockSyslogWriter) Clear() {
 }
 
 // Close releases resources. No other methods may be called after this.
-func (msw *MockSyslogWriter) Close() error {
```

Comment:
How did this struct ever work if it didn't ever compile for the one thing it was supposed to do?

It sounds like the error was something calling methods on a `MockSyslogWriter{}` instead of a `&MockSyslogWriter`?